include/asterisk/res_audiosocket.h

```diff
@@ -36,6 +36,25 @@ extern "C" {
 #include "asterisk/frame.h"
 #include "asterisk/uuid.h"
 
+
+enum ast_audiosocket_msg_kind {
+	/*! \brief Message indicates the channel should be hung up, direction: Sent only. */
+	AST_AUDIOSOCKET_KIND_HANGUP = 0x00,
+
+	/*! \brief Message contains the connection's UUID, direction: Received only. */
+	AST_AUDIOSOCKET_KIND_UUID   = 0x01,
+
+	/*! \brief Message contains a DTMF digit, direction: Received only. */
+	AST_AUDIOSOCKET_KIND_DTMF   = 0x03,
```

Comment:
Where is this documented? are you going to update the documentation page?

Reply:
This is documented here: https://github.com/CyCoreSystems/audiosocket/pull/24/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

We are waiting for this PR to be merged to merge the audiosocket PR as well.

Sure, I'll update [asterisk/documentation](https://github.com/asterisk/documentation)  too.

Reply:
I have requested the PR here: https://github.com/asterisk/documentation/pull/131